playbooks/roles/ocp_operator_deployment/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+# tasks file for ocp_operator_deployment
+- name: Verify that required variables are present
+  ansible.builtin.assert:
+    that:
+      - ocp_operator_deployment_operators is defined
+      - ocp_operator_deployment_operators | length > 0
+      - ocp_operator_deployment_version is defined
+    fail_msg: Required variables are not set or empty
+
+- name: Verify that stage repo required variables are present
+  loop: "{{ ocp_operator_deployment_operators }}"
+  when: item.catalog  == stage
```

Comment:
Maybe we should move this to the bottom of the task to be consistent with other tasks?